src/server_manager/web_app/ui_components/outline-share-dialog.ts

```diff
@@ -112,23 +112,56 @@ Polymer({
         ></p>
       </div>
       <div contenteditable="" id="selectableText" style="-webkit-text-size-adjust: 100%;">
-        <p>[[localize('share-invite')]]</p>
-
-        <p><a href$="{{s3Url}}">{{s3Url}}</a></p>
-        <p>-----</p>
-        <p>[[localize('share-invite-trouble')]]</p>
+        <p>[[localize('share-invite-intro')]]</p>
         <ol>
           <li>
-            [[localize('share-invite-copy-access-key')]]
-            <a href="{{accessKey}}">{{accessKey}}</a>
+            [[localize('share-invite-install-outline-step')]]
+            <ul>
+              <li>
+                <a href="https://play.google.com/store/apps/details?id=org.outline.android.client"
```

Comment:
It seems links are not preserved when sharing. We need to spell them out. See the suggestion in your doc.

Reply:
Formatting may be lost too. This is what I get when I copy the instructions hare:

You’re invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.

Download and install the Outline app for your device:
Android: [Play Store](https://play.google.com/store/apps/details?id=org.outline.android.client) - if blocked, try downloading the [APK file](https://s3.amazonaws.com/outline-releases/client/Outline-Client.apk) directly.
iOS: [App Store](https://itunes.apple.com/app/outline-app/id1356177741) 
Windows: [installer .exe](https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe) - once downloaded, double click to launch.
macOS: [App Store](https://itunes.apple.com/app/outline-app/id1356178125) ([Mirror](https://s3.amazonaws.com/outline-releases/client/Outline-Client.dmg)) 
Linux: [AppImage file](https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage) - once downloaded, [make it executable](https://docs.appimage.org/introduction/quickstart.html), then double click to launch.
Scan this QR code to have the service added to your Outline app:



If the QR code won’t work, copy the following access key. In the Outline app, tap the "+" sign on the top bar and paste the access key:


ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTptWk82N0dLODdzNXg@159.203.76.146:443/?outline=1#Canary


Press the "ADD SERVER" button to confirm.
A card will be added to your app. Tap "Connect" to get connected with the new service you added.

Reply:
Yeah, different clients support different paste "types." You can't paste a hyperlink with an alternative name directly into google docs, for example.

Reply:
How's this?

Reply:
You’re invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.
1) Download and install the Outline app for your device:
  - Android: https://play.google.com/store/apps/details?id=org.outline.android.client
  - iOS: https://itunes.apple.com/app/outline-app/id1356177741 
  - Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe - once downloaded, double click to launch.
  - macOS: https://itunes.apple.com/app/outline-app/id1356178125
  - Linux: https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage - once downloaded, make it executable, then double click to launch.
2) Copy the following access key: ss://[REDACTED]/?outline=1#Canary
3) In the Outline app, tap the "+" sign on the top bar and paste the access key.
4) Press the "ADD SERVER" button to confirm.
5) A card will be added to your app. Tap "Connect" to get connected with the new service you added.

Reply:
github formats this somewhat: 

You’re invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.
1) Download and install the Outline app for your device:
  - Android: https://play.google.com/store/apps/details?id=org.outline.android.client
  - iOS: https://itunes.apple.com/app/outline-app/id1356177741 
  - Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe - once downloaded, double click to launch.
  - macOS: https://itunes.apple.com/app/outline-app/id1356178125
  - Linux: https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage - once downloaded, make it executable, then double click to launch.
2) Copy the following access key: ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTptWk82N0dLODdzNXg@159.203.76.146:443/?outline=1#Canary
3) In the Outline app, tap the "+" sign on the top bar and paste the access key.
4) Press the "ADD SERVER" button to confirm.
5) A card will be added to your app. Tap "Connect" to get connected with the new service you added.

Reply:
github formats this slightly :/

You’re invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.
1) Download and install the Outline app for your device:
  - Android: https://play.google.com/store/apps/details?id=org.outline.android.client
  - iOS: https://itunes.apple.com/app/outline-app/id1356177741 
  - Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe - once downloaded, double click to launch.
  - macOS: https://itunes.apple.com/app/outline-app/id1356178125
  - Linux: https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage - once downloaded, make it executable, then double click to launch.
2) Copy the following access key: ss://[REDACTED]/?outline=1#Canary
3) In the Outline app, tap the "+" sign on the top bar and paste the access key.
4) Press the "ADD SERVER" button to confirm.
5) A card will be added to your app. Tap "Connect" to get connected with the new service you added.